geomstats/geometry/invariant_metric.py

```diff
@@ -179,11 +107,11 @@ def structure_constant(self, tangent_vec_a, tangent_vec_b, tangent_vec_c):
 
```

Comment:
Give a reference, even if it is wikipedia?

Reply:
Actually maybe another name should be found for this, because structure constants are specific to basis vectors whereas I use this formula for any vectors. It is just the projection of the bracket...